packages/rolldown/tests/plugin/plugin.test.ts

```diff
@@ -122,9 +122,7 @@ test('call transformContext error', async () => {
     })
     await build.write({})
   } catch (error: any) {
-    expect(error.message).toMatchInlineSnapshot(
-      `"Rolldown internal error: GenericFailure, RollupError: transform hook error"`,
-    )
+    expect(error.message).toMatchInlineSnapshot(`"Build failed"`)
```

Comment:
Hope the message could be more friendly in the future

Reply:
This comes from https://github.com/rolldown/rolldown/blob/7a4d36b04af6f750492f727518d05de67d64693c/crates/rolldown_binding/src/bundler.rs#L187.

I guess it could be improved indeed.

Reply:
I'll do it in other PR.